images/kindnetd/cmd/kindnetd/cni.go

```diff
@@ -75,7 +98,8 @@ const cniConfigTemplate = `
 				}
 			]
 		]
-		}
+		},
+		"mtu": {{ .Mtu }}
```

Comment:
can you please check if this renders correctly? 
I never remember if it needs a trailing comma

Reply:
JSON should not have a trailing comma.

Reply:
From my understanding, this `,` should be here. After looking at the example here https://github.com/containernetworking/plugins/tree/master/plugins/main/ptp#example-network-configuration

it looks like `mtu` is a field on the as the `ipam` dictionary.

Reply:
Yeah, it's ok, I was referring after MTU